### PR TITLE
Add logistics and prep materials to NW satellite page

### PR DESCRIPTION
**Pull request by leewujung**

### CODE REVIEW

**Comment by leewujung** [comment]
@dgumustel: I couldn't select you as a reviewer, but could you also review this PR? Thanks!

**Review by emiliom — APPROVED** [review]
Looks good

File: ohw22/seattle/index.md [hunk 1]
```diff
@@ -31,15 +31,49 @@ We welcome participants from all career stages and encourage applications from g
 3. We encourage students to communicate with the global virtual event and other satellite events to augment their OHW experience.
 
 
-**Logistics**
+### Location
+All activities will be in Ocean Sciences Building (OSB)  https://www.washington.edu/maps/#!/ocn on the University of Washington Seattle campus.
+- OSB 425 will be main location for tutorials and large group discussions
+- Other OSB locations can be leveraged for breakout discussions
```
Comment (dgumustel) [inline ohw22/seattle/index.md:37]: This section (ln 34-37) could be moved up to the location section a few lines up (ln 15), and it looks like the link to the UW map didn't hyperlink properly.

File: ohw22/seattle/index.md [hunk 2]
```diff
@@ -31,15 +31,49 @@ We welcome participants from all career stages and encourage applications from g
 3. We encourage students to communicate with the global virtual event and other satellite events to augment their OHW experience.
 
 
-**Logistics**
+### Location
+All activities will be in Ocean Sciences Building (OSB)  https://www.washington.edu/maps/#!/ocn on the University of Washington Seattle campus.
```
Comment (emiliom) [inline ohw22/seattle/index.md:35]:
```suggestion
All activities will be in [Ocean Sciences Building (OSB) https://uw.edu/maps/#!/ocn](https://uw.edu/maps/#!/ocn) on the University of Washington Seattle campus.
```

File: ohw22/seattle/index.md (hunk 1 above)
Comment (emiliom) [reply ohw22/seattle/index.md:37]: Good catch! That sounds good. Or we could just delete the current, higher-level Location section. Either way is fine with me.

Comment (leewujung) [reply ohw22/seattle/index.md]: Ok, I moved the location detail to the top level so that people can easily see it.

**Pull request merged**